drawings.go

```diff
@@ -55,11 +55,77 @@ func (b *Buffer) Line(x1, y1, x2, y2 int, color int) {
 	}
 }
 
+// Line - draw line using the Bresenham's algorithm
+func (b *Buffer) LineFast(x1, y1, x2, y2 int, color int) {
```

Comment:
let's replace the old versions with new ones

Reply:
That was my plan. I left the old one in in case you wanted to keep them. It's your project, it will always be your last call 😀

Reply:
Now I'm looking into implementing a good flood fill algo. More pull requests will follow!

Reply:
I actually checked the code of the current flood fill implementation and it's good. I'm looking into other raster primitives that might be handy. I was thinking Bézier curves (quadratical and cubical)